lib/nerves_time/rtc/abracon.ex

```diff
@@ -87,6 +88,16 @@ defmodule NervesTime.RTC.Abracon do
     end
   end
 
-  defp check_id({:ok, %{id: :ab_rtcmc_32768khz_ibo5_s3}}), do: :ok
+  @spec trickle_charge(I2C.bus(), I2C.address()) :: :ok | {:error, String.t()}
+  defp trickle_charge(i2c, address) do
```

Comment:
ℹ️ (optional, if upstreamed)

Does it make sense to make the trickle charging feature a public function so a developer can turn it on/off?